plugins/hls-selection-range-plugin/hls-selection-range-plugin.cabal

```diff
@@ -0,0 +1,59 @@
+cabal-version:      2.4
+name:               hls-selection-range-plugin
+version:            1.0.0.0
+synopsis:
+  HLS Plugin to support smart selection range
+
+description:
+  Please see the README on GitHub at <https://github.com/haskell/haskell-language-server#readme>
+
+license:            Apache-2.0
+license-file:       LICENSE
+author:
+  https://github.com/haskell/haskell-language-server/contributors
+
+maintainer:
```

Comment:
I think we're encouraging people to list themselves as author/maintainer in such places!

Reply:
Oh, that's great!